src/connectors/smart-sort/connectSmartSort.ts

```diff
@@ -0,0 +1,130 @@
+import { Connector } from '../../types';
+import { createDocumentationMessageGenerator, noop } from '../../lib/utils';
+
+export type SmartSortConnectorParams = {
+  relevancyStrictness?: number;
```

Comment:
I still think that having this default strictness could pose problems for detecting which state we are in. I think it should stay being set in the settings only